package.json

```diff
@@ -27,7 +27,6 @@
 		"react-markdown": "^9.0.1",
 		"react-router-dom": "^6.26.2",
 		"react-slick": "^0.30.2",
-		"react-snow-overlay": "^1.0.7",
```

Comment:
Can we keep the snow overlay package but just comment out the component for now?